### DISCUSSION TITLE
Onboarding Error

### DISCUSSION THREAD

**Discussion by kunalgithub** [ONDC-Official/developer-docs#39] [Q&A]
I am getting the error while calling `subscribe`

It hits the ondc-verif...html but throws this error. 

Which Python function should be used for signing the request id?  We used sign_response but it has not worked 

```
Generate SIGNED_UNIQUE_REQ_ID => ( Sign request_id using signing_private_key generated in step 7 )
```

```
 "error": {
        "type": "DOMAIN-ERROR",
        "code": "129",
        "path": null,
        "message": "https://kotak-ondc-uat.perfios.com : Domain verification is failed "
    }
```

**Comment by 92shreyansh** [comment]
@kunalgithub Please refer the thread https://github.com/ONDC-Official/developer-docs/discussions/31
